examples/ml/housing-prices/README.md

```diff
@@ -0,0 +1,247 @@
+# Boston Housing Prices
+
+This example creates a simple machine learning pipeline in pachyderm to train a regression model on the Boston Housing Dataset to predict the value of homes in Boston. The pipeline itself is written in Python, though a Pachyderm pipeline could be written in any language.
+
+<p align="center">
+  <img width="200" height="300" src="images/regression_pipeline.png">
+</p>
+
+The Pachyderm Pipeline will perform the following:
+
+1. Import the structured dataset (`.csv`) with pandas
+2. Perform some light data analysis with scikit-learn
+3. Train a regression model to predict housing prices
+4. Generate a learning curve and performance metrics to estimate the quality of the model
+
+Outline of this Example: 
+
+- [Boston Housing Prices](#boston-housing-prices)
+  - [Housing prices datasets](#housing-prices-datasets)
+  - [Getting Started](#getting-started)
+  - [Python code](#python-code)
+    - [Data Analysis](#data-analysis)
+    - [Train a regression model](#train-a-regression-model)
+    - [Evaluate the model](#evaluate-the-model)
+  - [Pachyderm Pipeline](#pachyderm-pipeline)
+    - [TLDR; Just give me the code](#tldr-just-give-me-the-code)
+    - [Step 1: Create input data repository](#step-1-create-input-data-repository)
+    - [Step 2: Create the regression pipeline](#step-2-create-the-regression-pipeline)
+    - [Step 3: Add the housing dataset to the repo](#step-3-add-the-housing-dataset-to-the-repo)
+    - [Step 4: Download files once the pipeline has finished](#step-4-download-files-once-the-pipeline-has-finished)
+    - [Step 5: Update Dataset](#step-5-update-dataset)
+
+## Housing prices datasets
+
+The housing prices dataset used for this example is a reduced version of the original, focusing on a subset of the features. It contains 3 input features (RM, LSTST, and PTRATIO) and the output, or target (MEDV) that we are learning to predict.
+|Feature| Description|
+|---|---|
+|RM |       Average number of rooms per dwelling|
+|LSTAT |    Percent lower status of the population|
+|PTRATIO |  Pupil-teacher ratio by town|
```

Comment:
I added some more information to the description of the features and what they roughly indicate. I can't say exactly why some of the features were originally selected, in the general case, but we selected the subset because they had correlation to the output class and made the problem run faster.